h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_model_parameters.R

```diff
@@ -0,0 +1,27 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+
+
+test.model.generic.drf <- function() {
+    data <- h2o.importFile(path = locate('smalldata/testng/airlines_train.csv'))
+    
+    # Binomial
+    cols <- c("Origin", "Distance")
+    original_model <- h2o.randomForest(x=cols, y = "IsDepDelayed", training_frame = data, validation_frame = data, nfolds = 3)
```

Comment:
Only random forest is tested. Ideally, all supported Generic MOJO models should be tested. Including this test into already existing pyunit and R tests would:

1. Reduce the load on test pipeline (another model trained, while we could check the presence of `_parms` in models already trained.
2. Test all supported models at once.